vtm-android/src/org/oscim/android/MapView.java

```diff
@@ -113,8 +119,9 @@ public MapView(Context context, AttributeSet attributeSet) {
         mMap = new AndroidMap(this);
 
         /* Initialize Renderer */
+        setEGLContextClientVersion(targetGLESVersion);
         setEGLConfigChooser(new GlConfigChooser());
-        setEGLContextClientVersion(2);
+        setEGLContextFactory(new GlContextFactory());
 
```

Comment:
Since use a `EGLContextFactory` to create an OpenGL ES context, the `setEGLContextClientVersion` can be omitted?

Also the order can be:
- `setEGLContextFactory`
- `setEGLConfigChooser`

Reply:
I think you're right. Context version is taken from factory.

As long these two are set before `setRenderer` the order doesn't matter.